component.yaml

```diff
@@ -8,11 +8,10 @@ inputs:
   isRequired: false
   title: Project Path
   description: The path to the directory that contains the definition of the project. Defaults to current directory.
-- key: "AC_TUIST_PROJECT_ONLY"
-  defaultValue: "false"
+- key: "AC_TUIST_VERSION"
   isRequired: false
```

Comment:
When I do not specify the path I get this error:
```
@@[command] tuist generate 
/Volumes/agent-disk/appcircle-runner/workflow_data/swouog4a.5qy/step_cp4ciqfw.25u/main.rb:23:in `runCommand': Manifest not found at path /Volumes/agent-disk/appcircle-runner/workflow_data/swouog4a.5qy (RuntimeError)
Consider creating an issue using the following link: https://github.com/tuist/tuist/issues/new/choose
	from /Volumes/agent-disk/appcircle-runner/workflow_data/swouog4a.5qy/step_cp4ciqfw.25u/main.rb:42:in `<main>'
Loading and constructing the graph
It might take a while if the cache is empty
```

Reply:
@boztopuz is it required or not?